NEWDB/Mongo/WNR.js

```diff
@@ -0,0 +1,114 @@
+const faker = require('faker');
+const fs = require('fs');
+const v8 = require('v8');
```

Comment:
Interesting to see that you are using node v8 module. Will be interested in learning how this works in our next meeting, since I never used it earlier